components/widget/match/summary/widget_match_summary_detailed_score.lua

```diff
@@ -0,0 +1,57 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Match/Summary/DetailedScore
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+
+---@class MatchSummaryDetailedScore: Widget
+---@operator call(table): MatchSummaryDetailedScore
+local MatchSummaryDetailedScore = Class.new(Widget)
+
+---@return Widget
+function MatchSummaryDetailedScore:render()
+	local flipped = self.props.flipped
+	local partialScores = Array.map(self.props.partialScores or {}, function(partialScore)
+		return HtmlWidgets.Td{
+			classes = {'brkts-popup-body-match-sidewins', partialScore.style},
+			children = partialScore.score
+		}
+	end)
+
+	return HtmlWidgets.Div{
+		css = {width = '70px', textAlign = 'center', direction = flipped and 'rtl' or 'ltr'},
+		children = HtmlWidgets.Table{
+			css = {lineHeight = '29px', float = flipped and 'right' or 'left'},
```

Comment:
why 29?
seems like a number we would usually not use
wasn't it said sometime to try to have px values devisable by 4?